Content.Shared/SimpleStation14/Species/Shadowkin/Components/ShadowkinComponent.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SimpleStation14.Species.Shadowkin.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class ShadowkinComponent : Component
+{
+    #region Random occurrences
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MaxedPowerAccumulator = 0f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MaxedPowerRoof = 0f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MaxedPowerRateMin = 45f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MaxedPowerRateMax = 150f;
+
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinPowerAccumulator = 0f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinPowerRoof = 0f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinPowerMin = 15f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinPowerMax = 60f;
```

Comment:
Decrease the max a bit, maybe 40?

```suggestion
    [ViewVariables(VVAccess.ReadWrite)]
    public float MinPowerMin = 15f;

    [ViewVariables(VVAccess.ReadWrite)]
    public float MinPowerMax = 40f;
```